syz-cluster/pkg/db/migrations/1_initialize.up.sql

```diff
@@ -0,0 +1,86 @@
+CREATE TABLE Series (
+    ID STRING(36) NOT NULL, -- UUID
+    ExtID STRING(128) NOT NULL, -- For LKML, it's a message ID of the series.
```

Comment:
Using ExtID as ID we'll remove one column and one index.

Reply:
That's an interesting question. Moved it to https://github.com/google/syzkaller/issues/5672.